test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/Package/Download/PackageDownloadRunnerTests.cs

```diff
@@ -661,6 +694,60 @@ public async Task RunAsync_WhenSourceMappingMapsToSecureLocalSource_DoesNotLogEr
         File.Exists(Path.Combine(installDir, $"{id.ToLowerInvariant()}.{version}.nupkg")).Should().BeTrue();
     }
 
+    [Fact]
+    public async Task RunAsync_WhenSourceMappingEnabled_PackageMapsToNoSource_Fails()
+    {
+        // Arrange
+        using var context = new SimpleTestPathContext();
+        PackageSource source = new("http://contoso.test/v3/index.json", "Insecure");
```

Comment:
Can we just use a local source here?

I'm not 100% sure how we're using a http test source either, so maybe we can clean that up.